source/publishing_data/06_configuring_metadata/standard_metadata.rst

```diff
@@ -0,0 +1,87 @@
+Standard metadata
+=================
+
+Standard metadata are common to all OpenDataSoft portal. They are the default metadata of any dataset. They should be configured in the Information tab of the edition interface of a dataset. Once the dataset is published, the standard metadata will be displayed in the Information tab of the dataset (front office).
+
+Refer to the table below to know more about the available metadata and how to configure them.
+
+.. list-table::
+  :header-rows: 1
+
+  * * Name
+    * Description
+    * Usage
+  * * Title
+    * Title of the dataset. Only metadata located above the edition tabs of the interface instead of in the Information tab.
+    * Write a dataset title in the textbox. All special characters are supported. Modifying the title does not change the technical identifier of the dataset.
+  * * Identifier
+    * Technical identifier of the dataset.
+    * 1. Click the "Modify identifier" button.
+      2. Modify the current identifier of the dataset.
+      3. Click the "Modify" button for the changes to be taken into account.
+
+      .. admonition:: Warning
```

Comment:
We could add another warning: the dataset_id cannot be changed if the dataset is published (we have the question on the support).